CODEOWNERS

```diff
@@ -28,13 +28,8 @@ src/dmd/target.d @ibuclaw @MartinNowak
 src/dmd/transitivevisitor.d @RazvanN7
 src/dmd/vcbuild @rainers
 
-# GitHub's implementation of the CODEOWNERS format is buggy, so this might not work
-src/dmd/*.h @ibuclaw
-
-src/dmd/* @WalterBright
-src/dmd/backend/* @WalterBright
-src/dmd/root/* @WalterBright
-src/dmd/tk/* @WalterBright
+# GitHub's implementation of the CODEOWNERS format is buggy, it doesn't allow src/dmd/*.h
```

Comment:
It didn't work here: https://github.com/dlang/dmd/pull/7627
According to [their guide](https://help.github.com/articles/about-codeowners/) `*.h` should work.